File: content/en/api/v1/webhooks-integration/GetWebhooksIntegrationCustomVariable.java
import com.datadog.api.v1.client.api.WebhooksIntegrationApi;

public class WebhooksIntegrationApiExample {

    public static void main(String[] args) {
        WebhooksIntegrationApi apiInstance = new WebhooksIntegrationApi();
        String customVariableName = customVariableName_example; // String | The name of the custom variable.
        try {
            WebhooksIntegrationCustomVariable result = apiInstance.getWebhooksIntegrationCustomVariable(customVariableName);
            System.out.println(result);
        } catch (ApiException e) {
            System.err.println("Exception when calling WebhooksIntegrationApi#getWebhooksIntegrationCustomVariable");
            e.printStackTrace();
        }
    }
}

File: content/en/api/v1/webhooks-integration/GetWebhooksIntegrationCustomVariable.rs
// Get a custom variable returns "OK" response
use datadog_api_client::datadog;
use datadog_api_client::datadogV1::api_webhooks_integration::WebhooksIntegrationAPI;

#[tokio::main]
async fn main() {
    let configuration = datadog::Configuration::new();
    let api = WebhooksIntegrationAPI::with_config(configuration);
    let resp = api
        .get_webhooks_integration_custom_variable("custom_variable_name".to_string())
        .await;
    if let Ok(value) = resp {
        println!("{:#?}", value);
    } else {
        println!("{:#?}", resp.unwrap_err());
    }
}
